src/autodev.ts

```diff
@@ -75,44 +95,41 @@ const autoDev = async (): Promise<void> => {
   await exec('git fetch')
   await exec(`git config --global user.email "${email}"`)
   await exec(`git config --global user.name "${user}"`)
-  await exec(`git checkout ${branch}`)
-  await exec(`git reset --hard origin/${base}`)
+  const commitDate = await execAndSlurp(
+    `git show -s --format='%ci' origin/${base}`
+  )
+
+  await exec(`git checkout ${base}`)
 
   if (pulls.length === 0) {
-    if (await hasDiff('HEAD', `origin/${branch}`)) {
-      await exec('git push -f')
-      info(
-        `🎉 No Pull Requests found. Pushed changes, because "${branch}" and "${base}" diverged.`
-      )
-    } else {
-      info('🎉 No Pull Requests found. Nothing to merge.')
-    }
-    return
+    info('🎉 No Pull Requests found. Nothing to merge.')
```

Comment:
There is no need anymore to duplicate the check for a diff here

Reply:
No, I would still go to the push, so when all pulls get the "dev" label removed, the autodev action will revert the dev branch to main